noxfile.py

```diff
@@ -396,6 +397,7 @@ def doctest(session: nox.sessions.Session):
             "third_party",
             "--ignore",
             "third_party/bigframes_vendored/ibis",
+            "bigframes/core/compile/polars",
```

Comment:
I suspect this is actually including polars rather than ignoring. Let's try

```suggestion
            "--ignore",
            "bigframes/core/compile/polars",
```